wiremock/mappings/ProbationOffenderSearchApi_SearchByNomsNumber.json

```diff
@@ -0,0 +1,156 @@
+{
+  "id": "05c36ca3-f66c-4ffe-917e-65aff9567af2",
+  "request": {
+    "method": "POST",
+    "urlPathPattern": "/search.*",
+    "bodyPatterns": [
+      {
+          "equalToJson": { "nomsNumber": "A1234AI" }
+      }
+  ]
+  },
+  "response": {
+    "headers": {
+      "Content-Type": "application/json"
+    },
+    "status": 200,
+    "jsonBody": [
+      {
+        "offenderId": 1,
+        "softDeleted": false,
+        "otherIds": {
+          "crn": "X320741",
+          "pncNumber": "2018/0123456X",
+          "croNumber": "SF80/655108T",
+          "nomsNumber": "A1234AI",
+          "previousCrn": "X10001"
+        },
+        "title": "Mr",
+        "firstName": "Aadland",
+        "middleNames": [],
+        "surname": "Bertrand",
+        "previousSurnames": [],
+        "dateOfBirth": "1978-01-06",
+        "gender": "Male",
+        "currentDisposal": "1",
+        "currentExclusion": false,
+        "currentRestriction": false,
+        "partitionArea": "some name",
```

Comment:
partitionArea comes from `offenderManagers`. It would be `"National Data"` in this case